pkg/cmd/clients_test.go

```diff
@@ -295,101 +361,258 @@ func TestMobileClientsCmd_TestDeleteClient(t *testing.T) {
 	}
 }
 
+func NewClientsOrDie(config *restclient.Config) (kubernetes.Interface, m.Interface, sc.Interface) {
+	// create the K8client
+	k8client, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		panic(err.Error())
+	}
+	scClientSet, err := sc.NewForConfig(config)
+	if err != nil {
+		panic(err.Error())
+	}
+
+	mobileClientSet, err := m.NewForConfig(config)
+	if err != nil {
+		panic(err.Error())
+	}
+	return k8client, mobileClientSet, scClientSet
+}
+
+func homeDir() string {
+	if h := os.Getenv("HOME"); h != "" {
+		return h
+	}
+	return os.Getenv("USERPROFILE") // windows
+}
+
 func TestMobileClientsCmd_TestCreateClient(t *testing.T) {
+	var kubeconfig *string
```

Comment:
@maleck13 @witmicko I'm trying to wrap my head around these tests but in this particular test I think we are creating a real client and I don't see us doing that anywhere else, any reason why?

Reply:
sorry this is irrelevant, forgot to delete it, had it in as a way into debugger